src/plugins/intel_gpu/src/graph/impls/ocl/tile.cpp

```diff
@@ -46,23 +46,31 @@ struct tile_impl : typed_primitive_impl_ocl<tile> {
 namespace detail {
 
 attach_tile_impl::attach_tile_impl() {
-    implementation_map<tile>::add(impl_types::ocl, tile_impl::create, {
-        std::make_tuple(data_types::i8, format::bfyx),
-        std::make_tuple(data_types::u8, format::bfyx),
-        std::make_tuple(data_types::i32, format::bfyx),
-        std::make_tuple(data_types::f16, format::bfyx),
-        std::make_tuple(data_types::f32, format::bfyx),
-        std::make_tuple(data_types::i8, format::bfyx),
-        std::make_tuple(data_types::u8, format::bfyx),
-        std::make_tuple(data_types::i32, format::bfyx),
-        std::make_tuple(data_types::f16, format::bfzyx),
-        std::make_tuple(data_types::f32, format::bfzyx),
-        std::make_tuple(data_types::i8, format::bfwzyx),
-        std::make_tuple(data_types::u8, format::bfwzyx),
-        std::make_tuple(data_types::i32, format::bfwzyx),
-        std::make_tuple(data_types::f32, format::bfwzyx),
-        std::make_tuple(data_types::f16, format::bfwzyx),
-    });
+    const auto types = {data_types::i8, data_types::u8, data_types::i32, data_types::f16, data_types::f32};
+    const auto formats = {
+            format::bfyx,
+            format::bfzyx,
+            format::bfwzyx,
+            format::b_fs_zyx_fsv16,
+            format::b_fs_zyx_fsv32,
+            format::b_fs_yx_fsv16,
+            format::b_fs_yx_fsv32,
+            format::bs_fs_yx_bsv16_fsv16,
+            format::bs_fs_yx_bsv32_fsv16,
+            format::bs_fs_yx_bsv32_fsv32,
+            format::bs_fs_zyx_bsv16_fsv32,
+            format::bs_fs_zyx_bsv16_fsv16,
+            format::bs_fs_zyx_bsv32_fsv32,
+            format::bs_fs_zyx_bsv32_fsv16
+    };
+
+    std::set<std::tuple<data_types, format::type>> keys;
+    for (const auto& t : types) {
+        for (const auto& f : formats) {
+            keys.emplace(t, f);
+        }
+    }
+    implementation_map<tile>::add(impl_types::ocl, tile_impl::create, keys);
```

Comment:
we already have overloaded method to have two params formats and types, so we don't need to manually introduce "keys" here

Reply:
Done